F-Droid can't build

ref: https://github.com/jonasoreland/runnerup/pull/1063

ref: https://gitlab.com/fdroid/fdroiddata/-/jobs/3737884838#L504

/LE: FYI https://gitlab.com/fdroid/fdroiddata/-/commit/00dd899901edf429bbad04de44451b325a68826e

Can you reopen?


`hrdevice` depends on ANT+ which is [not FOSS](https://github.com/jonasoreland/runnerup/issues/662), right?

I can't just delete its folder
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:lintVitalReportLatestRelease'.
> Could not resolve all task dependencies for configuration ':app:latestReleaseRuntimeClasspath'.
   > Could not resolve project :hrdevice.
     Required by:
         project :app
      > No matching configuration of project :hrdevice was found. The consumer was configured to find a runtime of a component, preferably optimized for Android, as well as attribute 'com.android.build.api.attributes.BuildTypeAttr' with value 'release', attribute 'com.android.build.api.attributes.ProductFlavor:all' with value 'latest', attribute 'com.android.build.api.attributes.AgpVersionAttr' with value '7.4.1' but:
          - None of the consumable configurations have attributes.
```

--- a/build.gradle
+++ b/build.gradle
@@ -20,7 +20,7 @@ project.ext {
     compileSdkVersion = 33 //Update Travis manually
     targetSdkVersion = 33
 
-    appcompat_version = "1.6.0"
+    appcompat_version = "1.6.1"
     annotation_version = "1.5.0"
     //Note: Later Play Services will require a rewrite of NodeApi.NodeListener
     googlePlayServicesVersion = '11.0.4'
@@ -30,8 +30,8 @@ project.ext {
     mockitoVersion = '3.11.2'
 
     //The Git tag for the release must be identical for F-Droid
-    versionName = '2.5.0.3'
-    versionCode = 321
+    versionName = '2.5.0.4'
+    versionCode = 322
     latestBaseVersionCode = 15000000
 
     // F-Droid builds only allow free software (wear dir deleted at builds)
@@ -43,7 +43,7 @@ project.ext {
     antPlusLibPath = "$rootDir/ANT-Android-SDKs/ANT+_Android_SDK/API"
     antPlusLibName = "antpluginlib_3-8-0"
     if (!allowNonFree || project.hasProperty('org.runnerup.hr.disableAntPlus') || !rootProject.file(antPlusLibPath+'/'+antPlusLibName+'.aar').exists()) {
-        antPlusLib = ""
+        antPlusLibName = ""
     }
 
     // ApplicationId (possibly with a .debug suffix) shared Android and Wear
@@ -61,7 +61,7 @@ project.ext {
 // AGPBI: {"kind":"warning","text":"Using flatDir should be avoided because it doesn't support any meta-data formats.","sources":[{}]}
 allprojects {
     repositories {
-        if (rootProject.ext.antPlusLibPath) {
+        if (rootProject.ext.antPlusLibName) {
             flatDir { dirs rootProject.ext.antPlusLibPath }
         }
     }

--- a/hrdevice/build.gradle
+++ b/hrdevice/build.gradle
@@ -27,7 +27,7 @@ android {
         minSdkVersion 15
         targetSdkVersion rootProject.ext.targetSdkVersion
 
-        if (rootProject.ext.antPlusLibPath) {
+        if (rootProject.ext.antPlusLibName) {
             buildConfigField 'Boolean', 'ANTPLUS_ENABLED', "true"
         } else {
             android.sourceSets.main.java.excludes = ['org/runnerup/hr/AntPlus.java']
@@ -44,7 +44,7 @@ repositories {
 dependencies {
     implementation "androidx.appcompat:appcompat:${rootProject.ext.appcompat_version}"
 
-    if (rootProject.ext.antPlusLibPath) {
+    if (rootProject.ext.antPlusLibName) {
         implementation(name: rootProject.ext.antPlusLibName, ext: 'aar')
     }
 }

